project/cmake/modules/FindVAAPI.cmake

```diff
@@ -29,6 +29,10 @@ elseif(VAAPI_INCLUDE_DIR AND EXISTS "${VAAPI_INCLUDE_DIR}/va/va_version.h")
   unset(vaapi_version_str)
 endif()
 
+if(VAAPI_VERSION_STRING VERSION_LESS 0.34)
+message(FATAL_ERROR "VAAPI minimum version 0.34 not found")
+endif()
+
 if(NOT VAAPI_FIND_VERSION)
   set(VAAPI_FIND_VERSION 0.38.0)
```

Comment:
I'll give it a try.

I've updated the PA module yesterday in a similar way and that's what we get there.
-- Could NOT find PULSEAUDIO: Found unsuitable version "1.1.0", but required is at least "2.0.0" (found /usr/lib/x86_64-linux-gnu/libpulse.so)
(http://jenkins.kodi.tv/view/Linux/job/LINUX-64/8478/consoleFull)